src/data/scripts/campaign/intel/VayraPersonBountyManager.java

```diff
@@ -281,14 +280,14 @@ public void advance(float amount) {
                 Global.getSector().addScript(new PersonBountyManager());
                 log.info("Re-adding the vanilla bounty manager, alas");
             }
-            if (getPersonBountyEventDataFromRepository().isParticipating(Factions.PIRATES)) {
-                getPersonBountyEventDataFromRepository().getParticipatingFactions().remove(Factions.PIRATES);
```

Comment:
this was the culprit. obviously I missed one direct `remove()` and since the Repository now returns an unmodifiable list that prevents ninja removals like this (which would eventually lead to inexplicable `ConcurrentModificationException`s anyway), the first guy to try it got slapped on his hands even though somehow I didn't step on it.

still, a legitimate bugreport.